### PR TITLE
support monorepos

### PR DESCRIPTION
**Pull request by sakari**
support monorepos using wildcards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,26 +10,27 @@ add following to your steps after the step running your test suite
     - name: annotate uncovered lines
       uses: smartlyio/collect-coverage-action
       with:
-        coverage-file: coverage/coverage-final.json
+        coverage-file: packages/*/coverage/coverage.json
```
Comment (sakari) [inline README.md:13]: the wildcard specifies the sub package name used in the reporting

File: yarn.lock [hunk 2]
```diff
@@ -26,6 +26,13 @@
...
+"@cspotcode/source-map-support@^0.8.0":
+  version "0.8.1"
+  resolved "https://registry.yarnpkg.com/@cspotcode/source-map-support/-/source-map-support-0.8.1.tgz#00629c35a688e05a88b1cda684fb9d5e73f000a1"
+  integrity sha512-IchNf6dN4tHoMFIn/7OE8LWZ19Y6q/67Bmf6vnGREv8RSbBVb9LPJxEcnwrcwX6ixSvaiGoomAUvu4YSxXrVgw==
+  dependencies:
+    "@jridgewell/trace-mapping" "0.3.9"
+
 "@eslint/eslintrc@^1.3.0":
   version "1.3.0"
   resolved "https://smartly.af/artifactory/api/npm/npm/@eslint/eslintrc/-/eslintrc-1.3.0.tgz#29f92c30bb3e771e4a2048c95fa6855392dfac4f"
```
Comment (imarkkinen) [inline yarn.lock:38]: Had to nuke the file as there are references to smartly artifactory to get the packages install. Also installed @types/glob to get build working

**Review by imarkkinen — APPROVED** [review]
LGTM! Was able to run the test and results look sane. Ran into two minor things while installing deps.

File: yarn.lock (hunk 2 above)
Comment (sakari) [reply yarn.lock]: weird. I recreated the yarn.lock and now it points to yarnpkg which probably is better

**Pull request merged**